src/Storefront/Resources/views/storefront/utilities/form-validation-config.html.twig

```diff
@@ -1,9 +1,9 @@
 {% block form_validation_config %}
     {% set validationMessages = {
-        required: "error.VIOLATION::IS_BLANK_ERROR"|trans|sw_sanitize,
-        email: "error.VIOLATION::INVALID_EMAIL_FORMAT_ERROR"|trans|sw_sanitize,
-        confirmation: "error.VIOLATION::NOT_EQUAL_ERROR"|trans|sw_sanitize,
-        minLength: "error.VIOLATION::TOO_SHORT_ERROR"|trans|sw_sanitize,
+        required: 'error.VIOLATION::IS_BLANK_ERROR'|trans|sw_sanitize,
```

Comment:
@marcelbrode does this still work with your changes for escaping snippet `'`?

Should still be ok because of `|json_encode`